docs/changelog_link.rst

```diff
@@ -1,3 +0,0 @@
-Changes
```

Comment:
I guess I'm not super attached to it, but what's the reasoning behind removing the changelog from the docs?

Reply:
We'd discussed it not being needed in readthedocs, so I left it out after initially changing from m2r -> recommonmark

Reply:
I am on board with removing the change log, but make sure the API docs still work in readthedocs when changing any of the rendering engines